@kiva/kv-components/tests/fixtures/mockFeedActivityData.js

```diff
@@ -22,8 +22,133 @@ const comments = {
 						data: {
 							text: 'another guest comment',
 						},
+						user: {
+							data: {
+								publicLenderId: 'Nathan2352',
+								name: 'Nathan',
+								image: 'https://www-0.development.kiva.org/img/s100/6b1a24092be3aaa22216874e644a4acf.jpg', // eslint-disable-line max-len
+							},
+						},
 						parent: '',
-						latest_children: {},
+						latest_children: {
+							like: [
+								{
+									created_at: '2024-02-28T20:04:21.445272Z',
+									updated_at: '2024-02-28T20:04:21.445272Z',
+									id: '290a6c07-143a-416e-b016-ac0d9ac8ad91',
+									user_id: '5314401',
```

Comment:
Are these real user IDs in this new data? If so, let's change them to random IDs.

Reply:
We are not using them, so I'll remove.